src/components/CongratsModal/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { animated, useSpring } from 'react-spring';
+
+interface CongratulationsModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const CongratulationsModal: React.FC<CongratulationsModalProps> = ({
+  isOpen,
+  onClose,
+}) => {
+  const animation = useSpring({
+    opacity: isOpen ? 1 : 0,
+    transform: isOpen ? 'translateY(0)' : 'translateY(-50%)',
+  });
+
+  if (!isOpen) return null;
+
+  return (
+    <div className="fixed inset-0 flex items-center justify-center z-50 bg-black bg-opacity-50">
```

Comment:
Try to reuse the existing modal component for this

Reply:
this is not using, i'll remove this